src/components/ItemTable/ItemTableControls.tsx

```diff
@@ -38,8 +38,9 @@ const ItemTableControls = ({
   numItemsTotal = 0,
   filtersAreApplied = false,
 }: ItemTableControlsProps) => {
-  const viewAllLoadingMessage =
-    bib.getItemsViewAllLoadingMessage(filtersAreApplied)
+  const viewAllLoadingMessage = `Loading all ${
+    filtersAreApplied ? "matching" : numItemsTotal
+  } items. This may take a few moments...`
```

Comment:
`getItemsViewAllLoadingMessage` is only used here, why not change it instead of moving the functionality here? Or at least delete the declaration from the Bib model.

Reply:
Good catch, i removed it in the Bib model since it's only used here, but can add it back if you'd like greater separation of logic from presentation.

Reply:
Either approach is ok but I brought it up just as a question.